added release notes for 10.14.0.330 (#186)
SoftwareAG/c8y-releasenotes: content/release-10-14-0/devices-10-14-0-bundle/10-14-0-330.md

---
weight: 19
title: Release 10.14.0.330
layout: bundle
---

<!--10.14.0.325 - 10.14.0.330-->

### Fixes

<div><table ><colgroup>
<col style="width: 15%;"><col style="width: 70%;"><col style="width: 15%;"></colgroup>
<thead><tr>
<th>
Component</th>
<th>
Description</th>
<th>
Issue</th>
</tr>
</thead><tbody>

<tr>
<td>Loriot</td>

> <td>Events created by the Loriot agent when sending uplink requests will now be considered as device requests, providing connectivity status and availability monitoring.</td>

Reviewer: ```suggestion
<td>Events created by the Loriot agent when sending uplink requests are now considered device requests and provide connectivity status and availability monitoring.</td>
```
